### PR TITLE
Remove Sample output and file uids

### PR DESCRIPTION
**Pull request by lgnashold**

### CODE REVIEW

File: vals/graphql_client/__init__.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
     RunStatus,
```
Comment (lgnashold) [inline vals/graphql_client/__init__.py:26]: Changes in `graphql_client` are autogenerated and probably be safely ignored.

File: examples/example_scripts/sdk_example_with_create.py [hunk 2]
```diff
@@ -10,24 +10,25 @@
...
+#             "checks": [
+#                 {"operator": "grammar"},
+#                 {"operator": "includes", "criteria": "Thomas Jefferson"},
+#             ],
+#         }
+#     ],
+# }
+
+# # This function creates a suite on the platform
+# suite_id = create_suite(suite_data=suite_data)
```
Comment (alexpattison) [inline examples/example_scripts/sdk_example_with_create.py:30]: ?

File: examples/v2/run_examples.py [hunk 3]
```diff
@@ -10,24 +10,28 @@
...
+    # suite = Suite(
+    #     title="Test Suite",
+    #     global_checks=[Check(operator="grammar")],
+    #     tests=[
+    #         Test(
+    #             input_under_test="What is QSBS?",
+    #             checks=[Check(operator="equals", criteria="QSBS")],
+    #         ),
+    #     ],
+    # )
```
Comment (alexpattison) [inline examples/v2/run_examples.py:22]: Is there a reason we're leaving this rather than deleting?

File: examples/v2/run_examples.py [hunk 4]
```diff
@@ -149,10 +153,10 @@ async def run_with_qa_pairs_and_context():
...
 async def all():
     await run_with_model_under_test()
-    await run_with_function()
-    await run_with_function_context_and_files()
-    await run_with_qa_pairs()
-    await run_with_qa_pairs_and_context()
+    # await run_with_function()
+    # await run_with_function_context_and_files()
+    # await run_with_qa_pairs()
+    # await run_with_qa_pairs_and_context()
```
Comment (alexpattison) [inline examples/v2/run_examples.py:159]: Same

**Pull request merged**